packages/cascara/src/atoms/Flex/FlexItem.js

```diff
@@ -35,7 +36,7 @@ const FlexItem = ({
   // Might need to expand this for other than strings later
   const isValidElement = typeof children !== 'string';
 
-  const element = isValidElement ? children : <div>{children}</div>;
+  const element = isValidElement ? children : <Role>{children}</Role>;
```

Comment:
@rociobaena this is a unique scenario where we do not need `Role`. We are only rendering a `div` in the scenario where we have a string by itself and we need to add other flex classes to it. If there is a dom element as the child, then we use that component itself. So the div is only being used for wrapping strings, in which case, we do not need any of the benefits of Role for using `as`.

Reply:
Brian, I think I got it (for the third time :P ) Also, In this case, it is not need because we should have the Role in the **Flex** component, that will create our root component